spec/requests/api/collections_spec.rb

```diff
@@ -134,6 +134,11 @@ def test_collection_bulk_query(collection, collection_url, klass, id = nil)
       test_collection_query(:hosts, hosts_url, Host, :guid)
     end
 
+    it 'queries NetworkRouters' do
```

Comment:
Similar question to https://github.com/ManageIQ/manageiq/pull/15524/files#r127473371